apps/marketing/public/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "",
+  "short_name": "",
```

Comment:
Consider setting `name` and `short_name` to `DeepDish — Data Management Framework` and `DeepDish`, respectively—or simply omit both for now.